src/hooks/useCacheQuery.ts

```diff
@@ -39,7 +39,7 @@ import {
 
 export default function useCacheQuery<
   TData = any,
-  TVariables = OperationVariables
+  TVariables extends OperationVariables = OperationVariables
```

Comment:
curious about what this change is for? (totally not blocking - really just curious)

Reply:
Actually, some research (after talking through that) lead me to this

# What `QueryResult` looks like on `main`
![image](https://github.com/CMSgov/easi-app/assets/15203744/1030a36d-d7ca-42ae-bcde-4434d851e2f7)

# What `QueryResult` looks like on this branch
![image](https://github.com/CMSgov/easi-app/assets/15203744/51e7b1d1-0d0b-4e16-a024-5df4883faf49)

# Synopsis

I think this means that because the `QueryResult`'s `TVariables' generic type (say that 5 times fast) was changed to have an `extends OperationVariables`, and we're ALSO returning that type, we needed to make our _own_ `TVariables` type also extend it! 

The way I see it, we're returning a generic type that changed, so we had to update our generic type to match.

Reply:
Nice, thanks for that clarification!

Reply:
Thanks Clay!! I appreciate the info! Awesome information